lib/net/ssh/gateway/version.rb

```diff
@@ -0,0 +1,7 @@
+module Net
+  module Ssh
+    module Gateway
+      VERSION = "1.2.0"
```

Comment:
```Ssh``` should be ```SSH``` and VERSION is likely 2.0.0 as the removal of Net::SSH::Gateway::Version is a non semver change